db/warehouse/migrate/20230804124734_update_financial_tables.rb

```diff
@@ -0,0 +1,14 @@
+class UpdateFinancialTables < ActiveRecord::Migration[6.1]
+  def change
+    # These tables are empty at this time, so the following is safe
+    StrongMigrations.disable_check(:rename_column)
+    rename_column :financial_clients, :do_you_have_a_voucher, :does_the_client_have_a_tenant_based_housing_voucher
```

Comment:
Could we push these through a map as we do w/ the EPIC importers instead of using their column labels? I'm worried that they will change the headers, and we will have to migrate the DB to accept their files?

Reply:
We can, but I'm not doing that now, we're hoping the data shape settles (or is settled).  Since this is only the second iteration, it feels like we should still be tracking to the state of things on the other side.